src/containers/contact/Contact.js

```diff
@@ -1,32 +1,53 @@
 import React from "react";
 import "./Contact.css";
 import SocialMedia from "../../components/socialMedia/SocialMedia";
-import {contactInfo} from "../../portfolio";
+import { contactInfo } from "../../portfolio";
 import { Fade } from "react-reveal";
 
 export default function Contact() {
-    return (
-        <Fade bottom duration={1000} distance="20px">
-        <div className="main contact-margin-top" id="contact">
-            <div className="contact-div-main">
-                <div className="contact-header">
-                    <h1 className="heading contact-title">{contactInfo.title}</h1>
-                    <p className="subTitle contact-subtitle">{contactInfo.subtitle}</p>
+  return (
+    <Fade bottom duration={1000} distance="20px">
+      <div className="main contact-margin-top" id="contact">
+        <div className="contact-div-main">
+          <div className="contact-header">
+            <h1 className="heading contact-title">{contactInfo.title}</h1>
+            <p className="subTitle contact-subtitle">{contactInfo.subtitle}</p>
 
-                    <div className="contact-text-div">
-                        <a className="contact-detail" href={"tel:" + contactInfo.number}>{contactInfo.number}</a>
-                        <br/><br/>
-                        <a className="contact-detail-email"
-                           href={"mailto:" + contactInfo.email_address}>{contactInfo.email_address}</a>
-                        <br/><br/>
-                        <SocialMedia/>
-                    </div>
-                </div>
-                <div className="contact-image-div">
-                    <img alt="Saad Working" src={require("../../assets/images/contactMail.png")}></img>
-                </div>
+            <div className="contact-text-div">
+              <a className="contact-detail" href={"tel:" + contactInfo.number}>
+                {contactInfo.number}
+              </a>
+              <br />
+              <br />
+              <a
+                className="contact-detail-email"
+                href={"mailto:" + contactInfo.email_address}
+              >
+                {contactInfo.email_address}
+              </a>
+              <br />
+              <br />
+              <SocialMedia />
             </div>
+          </div>
+          <div className="contact-image-div">
+            <picture>
+              <source
+                type="image/webp"
+                srcset={require("../../assets/images/contactMail.webp")}
+              />
+              <source
+                type="image/png"
+                srcset={require("../../assets/images/contactMail.png")}
+              />
+              <img
+                alt="Saad Working"
+                src={require("../../assets/images/contactMail.png")}
+              ></img>
+            </picture>
+          </div>
         </div>
-        </Fade>
-    );
+      </div>
+    </Fade>
+  );
```

Comment:
Avoid adding file formatting to your commits in the future. It makes it tough to review if there's a lot of files changed.